.pre-commit-config.yaml

```diff
@@ -0,0 +1,61 @@
+repos:
+  # other fast helpful checks
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.3.0
+    hooks:
+      - id: check-yaml
+        args: [--allow-multiple-documents]
+      - id: check-json
+      # - id: end-of-file-fixer
+      # - id: trailing-whitespace
+      #   exclude: ^manifests/gcp_marketplace/test/*
+      # - id: debug-statements
+      # - id: check-merge-conflict
+      # - id: name-tests-test
+      # - id: double-quote-string-fixer
+      # - id: no-commit-to-branch
+      #   args: [--branch, master]
+  # required formatting jobs (run these last)
```

Comment:
Do we want to keep hooks that will be only applicable to the Katib code base ?

Reply:
Which hooks are you referring to?

Reply:
Are we planning to use these hooks for Katib in the future ?
- https://github.com/pre-commit/mirrors-yapf
- https://github.com/pycqa/docformatter
- https://github.com/hadialqattan/pycln